### ISSUE TITLE
cifar10_train.yaml `me_max` parameter

### ISSUE THREAD

**Issue by jmarrietar** [facebookresearch/suncet#13]
Hi @MidoAssran, I have a question regarding the default CIFAR-10 YAML config file. 

For CIFAR10 shouldn't the regularizer to be set to False?  `me_max: false` ?. When I use True (Default parameter in `cifar10_train.yaml` file the loss is negative all the training). 

When I put it to **False** it matches more with the loss reported for CIFAR-10 in the paper 
![Screen Shot 2021-06-03 at 10 51 42 AM](https://user-images.githubusercontent.com/4519569/120674877-27dfa100-c45a-11eb-96c9-876bbc3860cc.png)

**Comment by MidoAssran** [comment]
hi @jmarrietar , what's plotted is only the cross-entropy portion of the loss (not the loss + me-max), even though we had `me_max: true`. The me-max regularizer (negative entropy of average prediction) tends to stay close to its minimum value during training so it's not that interesting to look at. Turning it off may not make much of a difference, but we left it on for the CIFAR10 experiments.  Hope that clarifies things!

**Comment by jmarrietar** [comment]
Yes, That makes Sense @MidoAssran  Thank you!.
